content/chapter-destinations/destination-loki/_index.md

```diff
@@ -0,0 +1,136 @@
+---
+title: "loki: Grafana Loki"
+weight:  3050
+---
+<!-- This file is under the copyright of Axoflow, and licensed under Apache License 2.0, except for using the Axoflow and AxoSyslog trademarks. -->
+
+Available in {{% param "product.abbrev" %}} version 4.4 and later.
+
+The `loki()` destination sends your log data to [Grafana Loki](https://grafana.com/docs/loki/) via gRPC, using the same message format documented for the [Grafana Loki HTTP endpoint](https://grafana.com/docs/loki/latest/reference/api/#push-log-entries-to-loki).
+
+Sample configuration:
+
+```shell
+loki(
+    url("localhost:9096")
+    labels(
+        "app" => "$PROGRAM",
+        "host" => "$HOST",
+    )
+
+    workers(16)
+    batch-timeout(10000)
+    batch-lines(1000)
+);
+```
+
+## Prerequisites
+
+<!-- FIXME  -->
+
+## Options
```

Comment:
`template()`: Specifies a template defining the logformat to be used in the destination.

Default: `$ISODATE $HOST $MSGHDR$MSG`